test/brsTypes/components/RoSGNode.test.js

```diff
@@ -441,6 +441,19 @@ describe("RoSGNode", () => {
             });
         });
 
+        describe("hasfield", () => {
+            it("returns presence of a field", () => {
+                let node = new RoSGNode([{ name: new BrsString("foo"), value: new Int32(17) }]);
+
+                let hasField = node.getMethod("hasfield");
+                let result1 = hasField.call(interpreter, new BrsString("foo"));
+                expect(result1).toEqual(new BrsBoolean(true));
```

Comment:
Also for future reference, this could be a tick simpler with `expect(result1).toBe(BrsBoolean.True)`.  Because there's only the two possible `BrsBoolean` values, they're exposed as static members of `BrsBoolean` for easy access!